projects/elements/src/lib/lazy-elements/lazy-elements-loader.service.ts

```diff
@@ -32,8 +34,16 @@ export class LazyElementsLoaderService {
     });
   }
 
+  setGlobalOptions(options: LazyElementModuleGlobalOptions) {
+    this.options = options;
+  }
```

Comment:
Global options can be injected instead of added by the method as they are defined only once